crawler/plugins/environments/icp_environment.py

```diff
@@ -47,7 +47,9 @@ def get_container_namespace(self, long_id, options):
                 # for reg crawler
                 if regpod_pattern.search(podname):
                     # expected repotag is "repository/k8s-ns/imagename:tag"
+                    # other format is rejected
```

Comment:
why we reject other formats ? 
Also, if the repotag is "repository/k8s-ns/user-ns/imagename:tag", can we handle that ?